### DISCUSSION TITLE
DAI 2023-2024 - My first IntelliJ + Maven project - @AlexandreIorio

### DISCUSSION THREAD

**Discussion by AlexandreIorio** [heig-vd-dai-course/heig-vd-dai-course#45] [Show and tell]
https://github.com/AlexandreIorio/04_practical

**Comment by ludelafo** [comment]
Thank you!

I guess the project is private as I can't have access. Feel free to invite @hadrylouis and me to your project so we can have a look to it.

**Comment by AlexandreIorio** [comment]
I think everything is ok now

**Reply by ludelafo** [reply]
Thank you!

I see you have an empty README file. I would fill it to say a few words on what the project is about and how to build and run the app.

Also, in the `pom.xml`, you only have to keep one plugin (`maven-shade-plugin`), not both (`maven-jar-plugin` + `maven-shade-plugin`). We did update the documentation about this to avoid any confusion.

Feel free to fill the poll in the GitHub Discussion regarding this chapter: https://github.com/orgs/heig-vd-dai-course/discussions/3.

You can do the same with the chapter Git, GitHub and Markdown: https://github.com/orgs/heig-vd-dai-course/discussions/2.
